habitat-lab/habitat/tasks/rearrange/sub_tasks/nav_to_obj_task.py

```diff
@@ -54,6 +54,8 @@ def __init__(self, *args, config, dataset=None, **kwargs):
         self.robot_start_rotation = None
 
         self._min_start_distance = self._config.min_start_distance
+        self._goal_type = config.goal_type
```

Comment:
What does `goal_type` mean and what values can it take?

Reply:
this is supposed to distinguish ovmm end-to-end task from the other nav skills. Testing a change where end-to-end task uses a separate ovmm_task

Reply:
Let me see if I can get this done before the PR is merged. Otherwise, will handle this later as discussed.